packages/tools/fluid-runner/README.md

```diff
@@ -1,10 +1,16 @@
 # @fluidframework/fluid-runner
 
-This package contains utility for running various functionality inside a Fluid Framework environment.
+This package contains utility for running various functionality inside an offline Fluid Framework environment.
+Currently, the only function of the fluid-runner tool is `exportFile`. The tool may be expanded for futher functionality.
 
 ## Export File
 
-Allows some execution to be made on a container given a provided ODSP snapshot.
+Loads a Fluid container given a provided ODSP snapshot and code loader bundle, and allows some offline execution to be made on this container. The meaning of "execution" is generalized and it is up to consumer of this tool to determine what it means for their specific scenario.
```

Comment:
From the samples below it's not clear to me how the execution is customized - do I need to bake that into the loaded code?  Is there some script I can provide?  Is there some default behavior if I don't customize?